pycoal/environment.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
```

Comment:
@lewismc i notice the other source files aren't using `#!/usr/bin/env python`. do we need this, or should i remove it?